patroni/postgresql/validator.py

```diff
@@ -280,17 +280,18 @@ def _read_postgres_gucs_validators_file(file: str) -> Dict[str, Any]:
 def _load_postgres_gucs_validators() -> None:
     """Load all Postgres GUC validators from YAML files.
 
+    Load validators from YAML files that were found.
+
     Recursively walk through ``available_parameters`` directory and load validators of each found YAML file into
     ``parameters`` and/or ``recovery_parameters`` variables.
 
     Walk through directories in top-down fashion and for each of them:
         * Sort files by name;
-        * Load validators from YAML files that were found.
```

Comment:
Is it an expected change?

Reply:
Hmmn yes that is odd. Let me revert it and see if there were errors.

Reply:
Yep, unrelated. Reverted.